mirror/images.txt

```diff
@@ -9,18 +9,12 @@
 # ceph-csi devel
 docker.io/rook/ceph:v1.8.2	rook/ceph:v1.8.2
 
-
 # ceph-csi v3.5
-docker.io/ceph/ceph:v16		ceph/ceph:v16
-docker.io/rook/ceph:v1.6.2	rook/ceph:v1.6.2
-
-
-# ceph-csi v3.4
-docker.io/ceph/ceph:v16		ceph/ceph:v16
+quay.io/ceph/ceph:v16		ceph/ceph:v16
```

Comment:
please keep the values aligned (with tabs)

Reply:
not sure there is an inconsistency with tabs , however it looks to be rendering correctly to me https://github.com/ceph/ceph-csi/blob/511ae20d106ff5e3fd08ec2ed533e9b6e357c555/mirror/images.txt

Reply:
thanks, seems the GitHub webui/PR does not show it correctly for some reason.